csm/core/agent/api.py

```diff
@@ -169,17 +172,19 @@ def error_response(err: Exception, **kwargs) -> dict:
         if isinstance(err, CsmError):
             resp["error_code"] = err.rc()
             resp["message"] = err.error()
-            message_id = err.message_id()
-            if message_id is not None:
-                resp["message_id"] = err.message_id()
+            resp["message_id"] = err.message_id()
             message_args = err.message_args()
             if message_args is not None:
                 resp["error_format_args"] = err.message_args()
         elif isinstance(err, web_exceptions.HTTPError):
-            resp["message"] = str(err)
-            resp["error_code"] = err.status
+            resp["error_code"] = CSM_HTTP_ERROR
+            resp["message_id"] = str(err.reason)
+            resp["message"] = str(err.text)
```

Comment:
Please update,
resp["message_id"] = 'HTTPError'

add 'HTTPError' to const file and read this from const.

Reply:
updated